src/content/blog/ga_qas.md

```diff
@@ -0,0 +1,219 @@
+---
+title: An Introduction to GA-QAS
+author: Vu Tuan Hai
+day: 21
+month: 5
+year: 2024
+tags:
+  - quantum architecture search
+  - python
+  - qiskit
+  - genetic-algorithm
+---
+
+In various quantum optimization problems, choosing the right ansatz is a critical point that will affect the accuracy of the result. Many template ansatzes have been proposed, such as Graph Ansatz and EffecientSU2, … but they are limited in their applicability. Hence, we've created a search engine called **GA-QAS** (Genetic Algorithm for Quantum Architecture Search) to aid in the discovery of the right ansatz. In this post, we will guide you on how to use it efficiently.
+
+
+## Now, let’s start!
+
+# Step 0. Setup the environment
+
+Make sure that you installed Python 3 and the following packages:
+
+- qiskit 1.0+
+- matplotlib
+- tdqm
+
+We will use a Python package named <a href="https://github.com/vutuanhai237/qoop">$\langle \mathrm{qo}|\mathrm{op} \rangle$</a>, which is a core package for developing quantum circuit optimization frameworks. You can download it via `git` as:
+
+```
+git clone https://github.com/vutuanhai237/qoop.git
+```
+
+Note that you should put it on the same level as your Python/Jupyter Notebook file. The best is:
+
+```
+folder_name
+│   your_python_file.py
+|   your_jupyter_file.ipynb
+└───qoop
+│   └───evolution
+│       │   environment.py
+│       │   cross_over.py
+│       │   ...
+```
+
+# Step 1. Define problem (W state preparation)
+
+<figure>
+    <img src='/public/images/ga-qas/ga-qas_scheme.png' width="600" style="margin: auto" alt='' />
```

Comment:
As discussed in https://github.com/unitaryfund/unitary.fund/pull/483#discussion_r1651463677, image paths begin with `/images`

Reply:
Sorry for not checking the deployment, the images are fixed.